stdlib/public/stubs/Stubs.cpp

```diff
@@ -445,11 +445,27 @@ __mulodi4(di_int a, di_int b, int* overflow)
 #endif
 
 #if defined(__CYGWIN__) || defined(_WIN32)
+  #define strncasecmp _strnicmp
```

Comment:
Cygwin has `strncasecmp`, as does MinGW. I think you'll want `#ifdef _MSC_VER` here, but I'm out of my depth here.